app/HutchManager/Services/RquestPollingService.cs

```diff
@@ -151,7 +159,16 @@ public void SendToQueue(RquestQueryTask jobPayload,string queueName)
             var properties = channel.CreateBasicProperties();
             properties.Persistent = true;
             
-            byte[] body = Encoding.Default.GetBytes(JsonConvert.SerializeObject(jobPayload) );
+            if (await _featureManager.IsEnabledAsync(FeatureFlags.UseROCrates))
+            {
+              ROCratesQuery roCratesQuery = new QueryTranslator.RquestQueryTranslator().Translate(jobPayload);
+              body = Encoding.Default.GetBytes(JsonSerializer.Serialize(roCratesQuery,DefaultJsonOptions.Serializer));
```

Comment:
explicitly commit to UTF8.
```suggestion
              body = Encoding.UTF8.GetBytes(JsonSerializer.Serialize(roCratesQuery, DefaultJsonOptions.Serializer));
```